awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx

```diff
@@ -143,6 +170,34 @@ class JobOutput extends Component {
     }
   }
 
+  async handleDeleteJob() {
+    const { job, history } = this.props;
+    try {
+      switch (job.type) {
+        case 'project_update':
+          await ProjectUpdatesAPI.destroy(job.idd);
```

Comment:
This should be `ProjectUpdatesAPI.destroy(job.id);`. Please also add a unit test that fails for this.

Reply:
Good catch - that is fallout from testing the deletion error modal.